### DISCUSSION TITLE
Feature request: Custom title for dashboard widgets

### DISCUSSION THREAD

**Discussion by jonahgreenthal** [getredash/redash#7233] [Ideas]
As far as I can tell, a dashboard widget's title is always just the name of the query. However, sometimes I would like to give the widget a different title than the query. This is particularly useful if I'm using the same query multiple times in the same dashboard (with different parameters).

**Comment by filip-jezek** [comment]
With constant parameters, one can change the very behavior of the query, while still having the same codebase. This is very useful. However, the the title does not really match. Additionally, the description of the query might be edited in the dashboard widget too to reflect those changes.
